File: Level-1/17.java
class Solution {
    public String solution(String s) {
        String answer = "";
        s = s.toLowerCase();
        String[] ss = s.split(" ", -1);
        for(int i = 0; i < ss.length; i++){
            for(int j = 0; j < ss[i].length(); j++){
                if(j % 2 == 0){
                    answer += (char)(ss[i].charAt(j) - 32);
                }else{
                    answer += ss[i].charAt(j);
                }
            }
            
            if(i != ss.length - 1){
                answer += " ";
            }
        }
        
        return answer;
    }
}

File: Level-1/17.js
function solution(s) {
    let answer = '';
    s.split(' ').forEach(v => {
        for(let i = 0; i < v.length; i++) {
            if(i % 2 === 0) {
                answer += v[i].toUpperCase();
            }else {
                answer += v[i].toLowerCase();
            }
        }
        answer += ' ';
    });
    return answer.substr(0, answer.length - 1);
}
